src/documentation/0038-node-make-http-requests/index.md

```diff
@@ -34,14 +34,36 @@ req.end()
 
 ## Perform a POST Request
 
+There are many ways to perform an HTTP POST request in Node.js, depending on the abstraction level you want to use.
+
+The simplest way to perform an HTTP request using Node.js is to use the [Axios library](https://github.com/axios/axios):
+
 ```js
-const https = require('https')
+const axios = require('axios')
 
-const data = new TextEncoder().encode(
-  JSON.stringify({
-    todo: 'Buy the milk 🍼'
+axios
+  .post('https://whatever.com/todos', {
+    todo: 'Buy the milk'
+  })
+  .then(res => {
+    console.log(`statusCode: ${res.status}`)
+    console.log(res)
   })
-)
+  .catch(error => {
+    console.error(error)
+  })
+```
+
+Axios requires the use of a 3rd party library.
+
+A POST request is possible just using the Node.js standard modules, although it's more verbose than the two preceding options:
```

Comment:
```suggestion
A POST request is possible just using the Node.js standard modules, although it's more verbose than the above:
```

I think this was in the original text. Perhaps I'm blind, but I can only see one option  👀

Reply:
👍 you are correct, must've been a typo on the other page that I've just copy pasted! Good spot 🙇